apps/arweave/src/ar_config.erl

```diff
@@ -693,6 +693,7 @@ parse_webhook([], Webhook) ->
 parse_webhook_events([Event | Rest], Events) ->
```

Comment:
Do we have any documention on webhooks and how to specify them?

If not maybe worth adding to ar.erl? Perhaps as a section at the end with an explanation for config-only options

it looks like we have *some* config-only documentation under the ar.erl `config_file` option (it describes the `semaphores` option)